### PR TITLE
BAEL-7269: How to Convert Date to unix Timestamp in Java?

### PR DESCRIPTION
**Pull request by azhwani**
+ Create test cases to test how to convert Date to Unix Timestamp

### CODE REVIEW

File: core-java-modules/core-java-date-operations-4/src/test/java/com/baeldung/datetounixtimestamp/DateToUnixTimeStampUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    void givenDate_whenUsingDateClass_thenConvertToUnixTimeStamp() throws ParseException {
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+        dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
+        Date givenDate = dateFormat.parse("2023-10-15 22:00:00");
+
+        assertEquals(1697407200000L, givenDate.getTime());
+    }
+
+    @Test
+    void givenDate_whenUsingCalendarClass_thenConvertToUnixTimeStamp() throws ParseException {
```
Comment (JavierPeris) [inline core-java-modules/core-java-date-operations-4/src/test/java/com/baeldung/datetounixtimestamp/DateToUnixTimeStampUnitTest.java:35]: It seems ParseException is never thrown.

Comment (azhwani) [reply core-java-modules/core-java-date-operations-4/src/test/java/com/baeldung/datetounixtimestamp/DateToUnixTimeStampUnitTest.java]: removed

File: core-java-modules/core-java-date-operations-4/pom.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <version>${joda-time.version}</version>
+        </dependency>
+    </dependencies>
+
+    <properties>
+        <joda-time.version>2.12.5</joda-time.version>
```
Comment (JavierPeris) [inline core-java-modules/core-java-date-operations-4/pom.xml:25]: Could we use 2.12.6?

Comment (azhwani) [reply core-java-modules/core-java-date-operations-4/pom.xml]: Done

**Pull request merged**